base/libgit2/libgit2.jl

```diff
@@ -215,6 +215,22 @@ end
 
 Set the `url` for `remote` for the git repository `repo`.
 The default name of the remote is `"origin"`.
+
+# Examples
+
+```julia
+julia> repo_path = joinpath("test_directory", "Example")
```

Comment:
I didn't do it that way because `test_directory` doesn't actually exist. I think this could, actually, become a doctest but I was lazy.